src/HeaderBar/Profile/ProfileHeader.js

```diff
@@ -40,36 +41,36 @@ ProfileEmail.propTypes = {
     children: propTypes.string,
 }
 
-const ProfileEdit = ({ children, contextPath }) => (
-    <a
-        href={`${contextPath}/dhis-web-user-profile/#/profile`}
-        data-test="headerbar-profile-edit-profile-link"
-    >
-        {children}
+const ProfileEdit = ({ children }) => {
+    const baseUrl = useConfig().baseUrl
+    return (
+        <a
+            href={`${baseUrl}/dhis-web-user-profile/#/profile`}
```

Comment:
Contrary to the `Apps.js`, this one adds a slash. I think I've seen it in other files also.
We could just copye the `joinPath` as mentioned above

Reply:
Good catch, this is incorrect